fixieai/cli/agent/commands.py

```diff
@@ -443,9 +443,7 @@ def serve(ctx, path, host, port, use_tunnel, reload, use_venv):
                 str(port),
                 "--factory",
             ]
-            + ["--reload"]
-            if reload
-            else [],
+            + (["--reload"] if reload else []),
```

Comment:
Weird that this is showing as a diff since this is already fixed in main

Reply:
synced with main and it's fixed now